src/content/docs/en/guides/deploy/github.mdx

```diff
@@ -130,40 +63,98 @@ export default defineConfig({
             uses: actions/deploy-pages@v4
     ```
 
-    :::note
-    The astro action takes a few optional inputs. These can be provided by uncommenting the `with:` line and the input you want to use.
-    :::
+     The astro action takes a few optional inputs. These can be provided by uncommenting the `with:` line and the input you want to use. Additionally, you will need to define any public environment variables. (See the [GitHub documentation on setting secrets](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#creating-configuration-variables-for-a-repository) for adding private environment variables.)
     
     :::caution
     The official Astro [action](https://github.com/withastro/action) scans for a lockfile to detect your preferred package manager (`npm`, `yarn`, `pnpm`, or `bun`). You should commit your package manager's automatically generated `package-lock.json`, `yarn.lock`, `pnpm-lock.yaml`, or `bun.lockb` file to your repository.
     :::
 
-2. (Optional) If you pass environment variables to your Astro project during local development or when previewing builds, you will need to define any public variables in the `deploy.yml` file in order for them to be processed when you deploy to GitHub Pages. (See the [GitHub documentation on setting secrets](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#creating-configuration-variables-for-a-repository) for adding private environment variables.)
+2. In your Astro config file `astro.config.mjs`, set [`site`](/en/reference/configuration-reference/#site) to the GitHub URL of your deployed site.
```

Comment:
Do we need both “Astro config file” AND “`astro.config.mjs`”? Given we get the code block with the filename below, I think we can drop it here:
```suggestion
2. In your Astro config file, set [`site`](/en/reference/configuration-reference/#site) to the GitHub URL of your deployed site.
```